plan/config.go

```diff
@@ -68,7 +76,8 @@ func ReadConfigFromEnviron() (*Config, error) {
 
 	config := &Config{
 		Reports:        reports,
-		CallTimeout:    time.Duration(callTimeout),
+		CallTimeout:    callTimeout*time.Second,
+		HostTimeout:    hostTimeout*time.Second,
```

Comment:
Instead of casting these to time.Second, let's just accept a time.Duration string and use the `time.ParseDuration` function to turn that into a proper `time.Duration`.

This way, the unit will be *in* the configuration:

```
CALL_TIMEOUT=1s
HOST_TIMEOUT=500ms
```